workflows/data_pipelines/spectacle/spectacle_dag.py

```diff
@@ -0,0 +1,72 @@
+from datetime import timedelta
+
+from airflow.decorators import dag, task
+from airflow.utils.dates import days_ago
+
+from dag_datalake_sirene.helpers import Notification
+from dag_datalake_sirene.workflows.data_pipelines.spectacle.spectacle_config import (
+    SPECTACLE_CONFIG,
+)
+from dag_datalake_sirene.config import EMAIL_LIST
+from dag_datalake_sirene.workflows.data_pipelines.spectacle.spectacle_processor import (
+    SpectacleProcessor,
+)
+
+default_args = {
+    "depends_on_past": False,
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "email": EMAIL_LIST,
+    "retries": 1,
+}
+
+
+@dag(
+    tags=["entrepreneur spectacle"],
+    default_args=default_args,
+    schedule_interval="0 16 * * *",
+    start_date=days_ago(8),
+    dagrun_timeout=timedelta(minutes=60),
+    params={},
+    catchup=False,
+    on_failure_callback=Notification.send_notification_tchap,
+    on_success_callback=Notification.send_notification_tchap,
+)
```

Comment:
`dag_id` is mising !

Reply:
By default it takes the function name, here `data_processing_entrepreneur_spectacle` but I can add it for consistency if you prefer of course.

Reply:
No that's okay. But could you delete it for the other data sources, already refactored as well, as well ?